machines/.gitignore

```diff
@@ -4,3 +4,4 @@ bsg_bladerunner_configuration.rom
 bsg_bladerunner_memsys.rom
 bsg_bladerunner_configuration.v
 bsg_bladerunner_pkg.vh
+bsg_manycore_machine.h
```

Comment:
Should have been in a previous PR. Let's just ignore that this exists...

Reply:
No - separate PR! :-p